public/Data/Detail.json

```
@@ -0,0 +1,90 @@
+{
+  "RESULT": {
+    "home_id": 1,
+    "detail_header": {
+      "deposit": "1억6000",
+      "exclusive_m2": 27.78,
+      "exclusive_pyeong": 8,
+      "month_totla_cost": "60만원",
```

```suggestion
      "month_total_cost": "60만원",
```
오타 있다고 백엔드분에게 알려주세요!